### PR TITLE
Crawler info

### PR DESCRIPTION
**Pull request by pandurx**
i added a 2nd user agent string condition (solr-crawler), to resolve infinite redirects that is being picked up by the gsa crawler

### CODE REVIEW

File: mod/b_extended_profile/views/default/profile/owner_block.php [hunk 1]
```diff
@@ -32,7 +32,7 @@
 // @todo: create a link to edit the user profile picture
 
 // just dont display anything with icons when gsa hits page
-if (strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') === false) {
+if (strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') === false || strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'solr-crawler') === false) {
```
Comment (Phanoix) [inline mod/b_extended_profile/views/default/profile/owner_block.php]: Wouldn't this always evaluate to true unless $_SERVER['HTTP_USER_AGENT'] is both a gsa AND a solr crawler?

Comment (pandurx) [reply mod/b_extended_profile/views/default/profile/owner_block.php]: Thanks.. Updated 👍

File: mod/custom_index_widgets/start.php [hunk 2]
```diff
@@ -8,7 +8,7 @@
 	 
     function custom_index_widgets_init() {
 
-if (elgg_is_active_plugin('gc_fedsearch_gsa') && ((!$gsa_usertest) && strcmp($gsa_agentstring,strtolower($_SERVER['HTTP_USER_AGENT'])) == 0) || strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') !== false ) {
+if (elgg_is_active_plugin('gc_fedsearch_gsa') && ((!$gsa_usertest) && strcmp($gsa_agentstring,strtolower($_SERVER['HTTP_USER_AGENT'])) == 0) && (strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') !== false || strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'solr-crawler') === false) ) {
```
Comment (Phanoix) [inline mod/custom_index_widgets/start.php]: Is this supposed to be  === here? It makes that part evaluate to true for regular browsers.

File: mod/wet4/views/default/discussion/replies.php [hunk 3]
```diff
@@ -6,7 +6,7 @@
  * @uses $vars['show_add_form'] Display add form or not
  */
 
-if (elgg_is_active_plugin('gc_fedsearch_gsa') && ((!$gsa_usertest) && strcmp($gsa_agentstring,strtolower($_SERVER['HTTP_USER_AGENT'])) == 0) || strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') !== false ) {
+if (strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') !== false || strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'solr-crawler') === false ) {
```
Comment (Phanoix) [inline mod/wet4/views/default/discussion/replies.php]: and here

File: mod/custom_index_widgets/start.php (hunk 2 above)
Comment (pandurx) [reply mod/custom_index_widgets/start.php]: If user-agent is gsa-crawler or user-agent is solr-crawler, do not display widgets

File: mod/wet4/views/default/discussion/replies.php (hunk 3 above)
Comment (pandurx) [reply mod/wet4/views/default/discussion/replies.php]: Updated

**Pull request merged**